prerender.js

```diff
@@ -0,0 +1,56 @@
+let puppeteer = require('puppeteer');
```

Comment:
Let's use const for non variadic values.
Also put as upper case the variables that are a entry point for a library.